.travis.yml

```diff
@@ -1,9 +1,10 @@
 language: php
 php:
+  - '7.0'
+  - '5.6'
```

Comment:
Heh. Good catch, no idea. 
Travis only seems to be running 5.6 once though, so thats cool